converters/fromZigbee.js

```diff
@@ -733,6 +734,19 @@ const converters = {
             return payload;
         },
     },
+    powerfactor: {
+        cluster: 'haElectricalMeasurement',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const payload = {};
+            if (msg.data.hasOwnProperty('powerFactor')) {
```

Comment:
This logic can be added to the already existing `electrical_measurement` converter.

Reply:
I adding in electrical_measurement .